### DISCUSSION TITLE
ImportError: DLL load failed while importing _msprime: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by JaneeshBansal** [tskit-dev/msprime#2142] [General]
Hi,

I am trying to install msprime on my laptop, but whenever I try to run code, which produces an SVG, I receive this error **ImportError: DLL load failed while importing _msprime: The specified module could not be found.**

I have a windows laptop and have been using conda to install the package. 

Could I please have help with the installation?

Thank you
Janeesh

**Comment by jeromekelleher** [comment]
Hi @JaneeshBansal :wave:

I think this may be because you're installing msprime into the root environment, which (I don't understand why) isn't a good idea with conda. See [this thread](https://github.com/tskit-dev/msprime/issues/2093#issuecomment-1195280043) for more info.

**Reply by JaneeshBansal** [reply]
Hi @jeromekelleher 

Thanks for the reply 

I have managed to solve the DLL issue with the thread you sent! The code runs but the SVG image is not produced. I have just been using the quick start code shown below. 

![image](https://user-images.githubusercontent.com/93221080/206422037-82bae833-5bd8-4be0-b6b3-c27dff07edb1.png)

Janeesh

**Reply by jeromekelleher** [reply]
Great!

This code is intended to run in a Jupyter notebook - you'd need to save it to a file if you're running it in a console. Something like

```python
ts.draw_svg(path="trees.svg")
```

**Reply by JaneeshBansal** [reply]
Thank you it works!
